src/pages/index.jsx

```diff
@@ -1,5 +1,6 @@
 import LoginPage from "./AuthPage/LoginPage";
 import SignupPage from "./AuthPage/SignupPage";
 import BookmarkedPage from "./BookmarkedPage";
+import ExplorePage from "./ExplorePage";
 import HomePage from "./HomePage";
-export { HomePage, LoginPage, SignupPage, BookmarkedPage };
+export { HomePage, LoginPage, SignupPage, BookmarkedPage, ExplorePage };
```

Comment:
you can also directly export on the same line.

`export { default as LoginPage } from "./AuthPage/LoginPage";`

Reply:
Learnt something new